codersdk/deployment.go

```diff
@@ -1215,10 +1216,20 @@ when required by your organization's security policy.`,
 			YAML:        "stackdriverPath",
 			Annotations: clibase.Annotations{}.Mark(annotationExternalProxies, "true"),
 		},
+		{
+			Name:        "Enable Terraform debug mode",
+			Description: "Allow administrators to enable Terraform debug output.",
```

Comment:
Is this closer to the intended behaviour?

```suggestion
			Description: "Allow administrators to enable Terraform debug output for template authors.",
```

Reply:
Actually, it is not. It is rather:

`Allow administrators to enable Terraform debug output for administrators.`

so I will stick to the current form.